devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -170,11 +190,19 @@ def updated() {
 def configure() {
 	log.debug "Configuring device ${device.getDataValue("model")}"
 
-	def cmds = zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x21, DataType.UINT8, 30, 21600, 0x01) +
-			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x21) +
-			zigbee.addBinding(zigbee.ONOFF_CLUSTER) +
-			readDeviceBindingTable() // Need to read the binding table to see what group it's using
+	def cmds = []
+
+	if (isSomfySituo()) {
+		cmds += zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x21, DataType.UINT8, 30, 21600, 0x01, ["destEndpoint":0xE8]) +
+				zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x21, ["destEndpoint":0xE8]) +
```

Comment:
I'd put the reads before the configureReporting so we get the current state more quickly

Reply:
ok, done.